### PR TITLE
Fixed TinyMCE disability issue 

### PR DESCRIPTION
**Pull request by mrsaicharan1**
Disable issue in TinyMCE menu items
Re-factored code to fix the issue.

This PR is in reference to #2245.

### CODE REVIEW

**Comment by ale-rt** [comment]
@mrsaicharan1, for the future you are welcome to push a new branch in this repository instead of making a fork

**Comment by mrsaicharan1** [comment]
@ale-rt Changes done and all tests have passed successfully. Thank you for guiding me through this issue!

File: Products/CMFPlone/patterns/tinymce.py [hunk 1]
```diff
@@ -100,12 +100,13 @@ def get_all_style_formats(self):
         }, {
             'title': 'Alignment',
             'items': self.get_styles(alignment_styles)
-        }, {
+        },{
```
Comment (ale-rt) [inline Products/CMFPlone/patterns/tinymce.py]: I do not want to be annoying but here we still have a missing space...

Comment (mrsaicharan1) [reply Products/CMFPlone/patterns/tinymce.py]: Not at all. Will take care of the issue.

**Pull request merged**